index.bs

```diff
@@ -2512,8 +2512,8 @@ is called on.
 
 	1. Let <var>o</var> be a new object of type <var>n</var>.
 
-	2. Let <var>option</var> be a dictionary of the type <a href="#dfn-associated-option-object">associated</a> to the interface
-		<a href="#dfn-associated-interface">associated</a> to this factory method.
+	2. Let <var>option</var> be a dictionary of the type [[#associated-option-object|associated]] to the interface
```

Comment:
Well, no, this isn't quite right either....

Reply:
Right, the `[[#foo]]` syntax is for linking to sections/headings.  (And `{{Foo}}` is for IDL constructs.)

This is just an ordinary definition you're linking to, so it uses ordinary autolinking - `<a>associated option object</a>` or the shorthand `[=associated option object=]`. Since you want custom text, it'll be either `<a lt="associated option object">associated</a>` or `[=associated option object|associated=]`.

Reply:
FWIW, I did try `[=associated option object|associated=]` and no link is created.  But using '<a lt="associated option object">associated</a>` works fine.